src/components/multi-select/multi-select--inline.html

```diff
@@ -0,0 +1,81 @@
+<div class="bx--multi-select bx--list-box bx--list-box--inline">
+  <div role="button" class="bx--list-box__field" tabindex="0" aria-label="open menu" aria-expanded="false" aria-haspopup="true">
+    <span class="bx--list-box__label">Label</span>
+    <div class="bx--list-box__menu-icon">
+      <svg fill-rule="evenodd" height="5" name="caret--down" role="img" viewBox="0 0 10 5" width="10" aria-label="Open menu">
+        <title>Open menu</title>
+        <path d="M10 0L5 5 0 0z"></path>
+      </svg>
+    </div>
+  </div>
+</div>
+<div style="margin-bottom: 2rem;"></div>
```

Comment:
We try to avoid adding elements for demo purposes as this HTML will be directly copied and pasted by our users and possibly also displayed on the website. It looks a bit funny without it, but that's okay! 🙂

Reply:
Ah okay, what's the convention for state variations? e.g. open vs closed. That's the only reason why I'm adding the separators is so that during development we could see both states.

Reply:
We would usually add javascript to show open/closed variations, but in this example, I would just keep it as is but remove the separator. We can still see both states, they will just be right next to each other which is fine. I'm guessing we will add functionality to this eventually.

Reply:
That works! 😄